.travis.yml

```diff
@@ -26,9 +26,10 @@ matrix:
       env: SYMFONY_VERSION='^3.4'
     - php: 7.3
       env: SYMFONY_VERSION='^4.2'
-
-before_install:
-  - phpenv config-rm xdebug.ini
```

Comment:
this should be kept for versions having xdebug. Change it to `phpenv config-rm xdebug.ini || true` to avoid making it fail when xdebug is not available.